public/js/graph/postsetup.js.jsx

```diff
@@ -0,0 +1,26 @@
+/*this was moved outside of setup.js*/
```

Comment:
jsx files are transformed in-browser, so if a js file need a function from a jsx file that wasn't transformed yet, it wouldn't find it.  By moving `$(document).read` into a jsx file, it would mean all the jsx files would be transformed before any code will be run.